lib/src/model.dart

```diff
@@ -4911,6 +4911,10 @@ class PackageGraph {
         if (needsPrecacheRegExp.hasMatch(d.documentationComment) &&
             !precachedElements.contains(d)) {
           precachedElements.add(d);
+          if (m is TopLevelVariable) {
```

Comment:
Can you add a comment explaining why `TopLevelVariable`s are special cased here?

Reply:
done